### DISCUSSION TITLE
设备N1，要装Docker，Bookworm 和 Bullseye 选那个兼容性最好

### DISCUSSION THREAD

**Discussion by trashblazer** [ophub/amlogic-s9xxx-armbian#1460] [Other] [answered]
Bookworm  刚刚冻结，Bullseye 是上一个稳定版，Docker Engine 也刚刚支持了 Debian Bookworm

--
总结：求新又求稳 😆

**Comment by ophub** [comment]
debian系以稳著称。
我觉的你要不装Bookworm的话，天天心里都痒痒。

**Reply by trashblazer** [reply, marked as answer]
太对了，Bookworm + kernel 6.1 来咯

**Reply by chmdir** [reply]
怎么从bullseye 升级到 bookworm呢, 只能重装么

**Comment by ophub** [comment]
能直接升级，换源升级，搜索下教程

**Reply by weakwisdom** [reply]
抱歉, 在其他讨论话题来提个别的问题
您曾说, armbian-software只有在bullseye版本才能选308安装pve, 308这个选项在上几个bookworm的固件中好像是不可见状态
但好像最新一期固件中, 这个308选项又出现了, 这是不是意味着当前版本固件又可以安装运行pve了?

**Reply by LSP0123** [reply]
pve官方没支持arm平台，现在用的都是第三方的。
pve8出来之前是只有bullseye才可以装的
pve8是基于bookworm的，所以可以装，但是都是没官方arm支持的
你可以参考下面的手动安装，对于arm64平台，写安装pve脚本也是一件不容易的事情
https://foxi.buduanwang.vip/virtualization/pve/1902.html/
